### DISCUSSION TITLE
Option to promote fields to the info sidebar

### DISCUSSION THREAD

**Discussion by dstoyanoff** [directus/directus#8460] [Feature Requests]
There are some use-cases, where certain fields make sense to be left outside of the main content. A perfect example for that is the "Status" field. Putting it in the form with the main content doesn't field "system" enough, so an idea was born in Discord with a suggestion from @benhaynes to allow promoting certain fields to the sidebar, instead of the main form. This would however require that the sidebar is open by default to minimize the confusion for the users.

**Comment by dimitrov-adrian** [comment]
I also like the idea, but such option per field could be bit of overkill, may be separate group interface, that will render itself in the sidebar could serve better. It also could have required title and icon.
Not sure how it will act for small screens, if put some important field or button there as the sidebar reachability button is not very obvious.

**Comment by br41nslug** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
